docs/userguides/platform.md

```diff
@@ -35,7 +35,49 @@ Building a container for your application can be an advanced topic, we have incl
 
 ## Building your Bot
 
-TODO: Add build process and describe `silverback build --autogen` and `silverback build --upgrade`
+To build your container definition(s) for your bot(s), you can use the `silverback build` command. This command searches your `bots` directory for python modules, then auto-generates Dockerfiles.
+
+For example, if your directory is structured as suggested in [development](./development), the `silverback build --generate` command would generate:
+
+As an example, if you have a bots directory that looks like:
+
+```bash
+silverback build --generate
+```
+
+```
+bots/
+├── botA.py
+├── botB.py
+├── botC.py
+```
+
+This method will generate 3 Dockerfiles:
+
+```
+.silverback-images/
+├── Dockerfile.botA
+├── Dockerfile.botB
+├── Dockerfile.botC
+```
+
+You can then build your image using:
+
+```bash
+silverback build
+```
+
+You can then push your image to your registry using:
+
+```bash
+docker push your-registry-url/project/botA:latest
+```
+
+TODO: The ApeWorX team has github actions definitions for building, pushing and deploying.
+
+If you are unfamiliar with docker and container registries, you can use the [github-action](https://github.com/ApeWorX/github-action) github actions repository to assist in these steps.
```

Comment:
We will need a separate one for silverback

Reply:
I'm not sure what you mean

Reply:
oh, when we build it, you mean?

Reply:
yes, whatever github action we suggest will not be the current apeworx one you have linked

Reply:
Should I leave a TODO here?

Reply:
yes